QgisModelBaker/libili2db/ili2dbutils.py

```diff
@@ -60,10 +60,10 @@ def get_ili2db_bin(tool, stdout, stderr, ili2db_tools):
         except NetworkError as e:
             stderr.emit(
                 QCoreApplication.translate('ili2dbutils',
-                                           'Could not download {tool_name}\n\n  Error: {error}\n\nFile "{file}" not found. Please download and extract <a href="{ili2db_url}">{ili2db_url}</a>'.format(
+                                           'Could not download {tool_name}\n\n  Error: {error}\n\nFile "{file}" not found. Please download and extract <a href="{ili2db_url}">{tool_name}</a>'.format(
```

Comment:
Nice commit.
Can you open a separate pull request for this one so we can merge it quickly (and avoid merge conflicts if it's adjusted in several pull requests)?
In case of technical problems with that, let me know and I can do the git cherry-pick for you.

Reply:
Yes. New pull request: #295